resources/memorytest/run.sh

```diff
@@ -38,61 +38,47 @@ echo "Deploying for $DISTRO with NGINX version $NGINX_DEPLOY_VERSION ..."
 #
 case $DISTRO in
 
-  'ubuntu18')
-    MODULE_FILE="ubuntu.18.04.ngx_curity_http_phantom_token_module_$NGINX_DEPLOY_VERSION.so"
-    MODULE_FOLDER='/usr/lib/nginx/modules'
-    NGINX_PATH='/usr/sbin/nginx'
-    CONF_PATH='/etc/nginx/nginx.conf'
-    ;;
-  
-  'ubuntu20')
-    MODULE_FILE="ubuntu.20.04.ngx_curity_http_phantom_token_module_$NGINX_DEPLOY_VERSION.so"
+  'alpine')
```

Comment:
I changed the ordering of Linux distros everywhere in code so that they are the same as the table in the main README.